drizzle.config.ts

```
@@ -87,6 +87,6 @@ export default defineConfig({
   schema: "./src/db/schema.ts",
   dialect: "postgresql",
   dbCredentials: {
-    url: env.DATABASE_URL,
+    url: process.env.DATABASE_URL!,
```

ESLint

[error] Unexpected use of process.env.

(node/no-process-env)